motion-of-no-confidence.md

```diff
@@ -0,0 +1,32 @@
+# Motion of no confidence
+
+The steering committee has [responsibilities][], and it's important
+that those responsibilities are enforceable.  For example, in the
+unlikely event that a committee member doesn't [attend 75% of the
+year's meetings][minimum-attendance], the other committee members and
+the Software Carpentry community at large should be able to evict the
+bum.
+
+If at any time the committee passes (following their usual voting
+rules) a motion of no confidence or the [Foundation
+membership][membership] submits a petition signed by over half of the
+members, early elections must be immediately scheduled.  The early
```

Comment:
On Thu, Mar 12, 2015 at 12:54:44PM -0700, Katy Huff wrote:
> > +If at any time the committee passes (following their usual voting
> > +rules) a motion of no confidence or the [Foundation
> > +membership][membership] submits a petition signed by over half of the
> > +members, early elections must be immediately scheduled.  The early
>
> Suggestion of more concise rephrasing:
>
> "An early election must be immediately scheduled if (1)the steering
> committee passes a motion of no confidence or (2) the Foundation
> Membership submits a petition signed by over half of the Foundation
> Members."

Adopted with some capitalization tweaks in ebae756
(motion-of-no-confidence.md: Itemize early election triggers,
2015-03-12).  I've explained my feelings on capitalization in the
commit message, but I'm happy to adopt whatever conventions the board
feels are appropriate.